store/sqlstore/link_metadata_store.go

```diff
@@ -59,20 +59,17 @@ func (s SqlLinkMetadataStore) Get(url string, timestamp int64) (*model.LinkMetad
 
 	err := s.GetReplica().SelectOne(&metadata,
 		`SELECT
-				*
-			FROM
-				LinkMetadata
-			WHERE
-				URL = :URL
-				AND Timestamp = :Timestamp`, map[string]interface{}{"URL": url, "Timestamp": timestamp})
+            *
```

Comment:
This is still inconsistent, you replaced the previous existing tabs with spaces, so... depending on your editor configuration you will see it properly indented, or wrongly indented. you should use tabs there. I would suggest the previous number of tabs minus 1.

Reply:
Sorry not sure what happened there, thanks for your help.